Documentation/GettingStarted.md

```diff
@@ -0,0 +1,371 @@
+# Getting Started
+
+This guide will use Pilot to step through building a basic iTunes Store Search application for iOS and macOS. By the end, the reader should have a good understanding of:
+
+- The core MVVM stack, how it may differ from other implementations of MVVM, and how it supports a unidirectional data flow.
+- How `ModelCollection` acts as the foundation for composable data collections and subsequent updates.
```

Comment:
"and subsequent updates" sounds a little off to my ear. Maybe

How `ModelCollection` acts as the foundation for composable data collections and handles updates.

Reply:
Done - good call.